tests/unit/controllers/show-project-test.js

```diff
@@ -27,4 +26,39 @@ module('Unit | Controller | show-project', function(hooks) {
       fitBounds() {},
     });
   });
+
+  test('hasBBLFeatureCollectionGeometry is null when geometry is null', function(assert) {
+    const controller = this.owner.lookup('controller:show-project');
+    assert.ok(controller);
+
+
+    controller.model = {
+      bbl_featurecollection: {
+        features: [{
+          type: 'Feature',
+          geometry: null,
+        }],
+      },
+    };
+
+    const hasGeometry = controller.hasBBLFeatureCollectionGeometry;
+
+    assert.equal(hasGeometry, null);
+  });
+
+  test('hasBBLFeatureCollectionGeometry is null when features is empty', function(assert) {
+    const controller = this.owner.lookup('controller:show-project');
+    assert.ok(controller);
+
+
+    controller.model = {
+      bbl_featurecollection: {
+        features: [{}],
```

Comment:
This is slightly different from `features: []`. In JavaScript, an empty object evaluates to truthy. So, I think you're not quite testing for the behavior we're talking about, which is "features is empty". Thoughts?

Reply:
Changed!